### PR TITLE
[IN-3056] Update PATH to sdkmanager on vs4mac stacks

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/xamarin/files/xamarin_profile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export BITRISE_XAMARIN_FOLDER_PATH="${HOME}/bitrise/xamarin"
+export ANDROID_SDK_VERSION="2.1"
```
Comment (ofalvai) [inline roles/xamarin/files/xamarin_profile:2]: nit: it's the version of the `cmdline-tools` package, not the whole SDK. And I wouldn't expose it to users.

Comment (fehersanyi-bitrise) [reply roles/xamarin/files/xamarin_profile:2]: oh, is it all right if I change it to `PACKAGE_VERSION` then? we need to define this somehow, and I didn't want to have it in the path as a magic number

Comment (fehersanyi-bitrise) [reply roles/xamarin/files/xamarin_profile:2]: or `CMDLINE_TOOL_VERSION`

Comment (ofalvai) [reply roles/xamarin/files/xamarin_profile]: `CMDLINE_TOOL_VERSION` looks perfect

**Review by nagypeterjob — DISMISSED** [review]

**Review by nagypeterjob — APPROVED** [review]

**Pull request merged**